packages/modeling/src/geometries/path2/concat.js

```diff
@@ -1,8 +1,9 @@
 const fromPoints = require('./fromPoints')
 const toPoints = require('./toPoints')
-
+const equals = require('../../maths/vec2/equals')
 /**
  * Concatenate the given paths.
+ * If both contain the same point at the junction, merge it into one.
```

Comment:
I think this is a comment for the code change, not for the API documentation.

Reply:
well, it is both :) ... may be good to leave it so end user knows the sideeffects of the concat.